content/intermediate/operations/understanding-changesets/index.md

```diff
@@ -60,7 +130,13 @@ Next, create your first change set:
 3. Choose **Next** again in both the **Specify stack details** and **Configure stack options** pages, and then choose **Create change set**.
```

Comment:
Please update line 128 to use `cfn-workshop-understanding-changesets` stack name